doc/interfaces/gibbs_unringing_flow.rst

```diff
@@ -0,0 +1,72 @@
+.. _gibbs_unringing_flow:
+
+===============
+Gibbs Unringing
+===============
+
+This tutorial walks through the steps to perform Gibbs unringing using DIPY.
+
+You can try this using your own data; we will be using the data in DIPY.
+You can check how to :ref:`fetch the DIPY data<data_fetch>`.
+
+---------------------------
+Suppress Gibbs Oscillations
+---------------------------
+
+Magnetic Resonance (MR) images are reconstructed from the Fourier coefficients
+of acquired k-space images. Since only a finite number of Fourier coefficients
+can be acquired in practice, reconstructed MR images can be corrupted by Gibbs
+artefacts, which is manifested by intensity oscillations adjacent to edges of
+different tissue types [1]_. Although this artefact affects MR images in
+general, in the context of diffusion-weighted imaging, Gibbs oscillations
+can be magnified in derived diffusion-based estimates [1]_, [2]_.
+
+We will use the ``tissue_data`` dataset in DIPY to showcase this denoising
+method. As with any other workflow in DIPY, you can also use your own data!
+
+You may want to create an output directory to save the resulting Gibbs ringing
+-free volume::
+
+    mkdir gibbs_ringing_output
+
+To run the Gibbs unringing method, we need to specify path to the input volumes.
```

Comment:
(...) we need to specify the path to the input data (...)

Reply:
Reworded in 3b616bb